src/main/java/arthur/timings/DateTime.java

```diff
@@ -0,0 +1,44 @@
+package arthur.timings;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Handles string conversion to Date and Time objects.
+ */
+public class DateTime {
+    private LocalDate date;
+    private LocalTime time;
+    private final String str;

```

Comment:
Can you explain what is this `str` used for?

Reply:
This just holds the string version of the time. At the moment it is redundant but wanted to use this in one of the features in the future.